### ISSUE TITLE
Edit the events from the calendar

### ISSUE THREAD

**Issue by Benbb96** [danvratil/FBEventSync#30]
One enhancement could be to be able to edit the event (description, remainders, color, inviting friends) directly from the calendar.

But I think it may be really difficult to do the events are not created on our own account.

**Comment by danvratil** [comment]
Precisely, the main problem here is that Android does not really offer that level of ACL granularity. I can restrict or allow access on calendar level, but not on event level, so I wouldn't be able to allow users to edit only events they created. The second problem is that I don't even have the information which event they created (although there's a way how to get the information).

Custom per-event reminders might be doable since that's a local thing. I haven't looked into it too closely yet and instead went with the application-wide reminders, which you can customize in the app settings.

You can also customize the color of each calendar in the app settings. There's no way to have per-event colors in Android, but you can assign different colors to calendars, so you can have a different color for the Attending and the Birthday calendars for example.

Regarding inviting friends, there is no proper way to get the list of all your friends from Facebook (Facebook will only show me your friends who are also using the app, which is not that many I guess :smiley:). But now that I think about it, I see a way how I could abuse the birthday calendar to use it as your friend list. To invite them, I could add "Invite your friends" link to the description that would open the app and would let you choose whom to invite.

I was also looking into implementing RSVP support (so you can change your own Going/Maybe/Not going status), but I couldn't get it to work in the default Android calendar app. I'll have to dig into it a bit more I guess, the documentation is somewhat limited :-)

Duuude, I thought the app was basically complete and now you totally overwhelmed me with so many new ideas! :smile:

**Comment by Benbb96** [comment]
Ha ha I know what you mean! That's the good point of open source projects, others can always bring you new ideas 😆 

Anyway, I took a look at your code, I don't understand kotlin language but it seems interesting! I use React Native to make mobile app but I should try this too. 

That's a really useful app, I know some of my friends would definitively use it, so keep it updated 😄 
I made my contribution as I could by translating your strings in French on the Hosted Weblate 😉

Just another point, when you create an event on your calendar, you can choose a place and it will add a nice picture on the header. I don't know if it could be possible to have the same place used by Google Map to obtain the picture than the one given by Facebook.

Again, that's just some details, I already like the fact that we can choose different colors and reminders for the event type 😄
